WindowsServerDocs/remote/remote-access/vpn/always-on-vpn/deploy/vpn-deploy-ras.md

```diff
@@ -181,7 +181,9 @@ In this section, you can configure Remote Access VPN to allow IKEv2 VPN connecti
     c. In **Start IP address**, enter the starting IP address in the range you want to assign to VPN clients.
 
     d. In **End IP address**, enter the ending IP address in the range you want to assign to VPN clients, or in **Number of addresses**, enter the number of the address you want to make available. If you're using DHCP for this subnet, ensure that you configure a corresponding address exclusion on your DHCP servers.
-
+>[!NOTE] for optimal network performance, The VPN Server  itself  should not  have  a network interface in the IPv4 subnet  where the Clients  are assigned  IPv4 addresses from.
```

Comment:
Needs revisions:
    > [!NOTE]  
    > For optimal network performance, the VPN server itself should not have a network interface in the same IPv4 subnet that assigns IPv4 addresses to the clients.  If the VPN server does have a network interface in that subnet, a broadcast or multicast that is sent to that subnet could cause a latency spike.